src/pages/DetailsPage.js

```diff
@@ -149,6 +152,13 @@ const DetailsPage = (props) => {
                                 </View>
                             ) : null}
                         </View>
+                        <MenuItem
+                            title={`${props.translate('common.message')}${details.displayName}`}
+                            icon={Expensicons.ChatBubble}
+                            onPress={() => Report.fetchOrCreateChatReport([props.session.email, details.login])}
+                            wrapperStyle={styles.breakAll}
+                            shouldShowRightIcon
+                        />
```

Comment:
This PR caused a regression https://github.com/Expensify/App/issues/18859. We should show the message button only if you are viewing another profile that is not yours. This was partially fixed by https://github.com/Expensify/App/pull/7228 but it did only check against the current session email and didn't cover the secondary logins.